source/jormungandr/jormungandr/scenarios/qualifier.py

```diff
@@ -228,3 +228,8 @@ def nonTC_crit(j_1, j_2):
 
 def duration_crit(j_1, j_2):
     return compare_minus(j_1.duration, j_2.duration)
+
+
+def get_ASAP_journey(journeys, req):
+    best_crit = arrival_crit if req["clockwise"] else departure_crit
```

Comment:
Not sure if it has an impact on performance but why not pass only this parameter instead of the whole request object?

Reply:
it won't change anything in term of performance, since the request is object and passed by reference :)